modules/lead/toolchain/rode.tf

```diff
@@ -0,0 +1,63 @@
+module "ca-issuer" {
```

Comment:
tf style nitpick: names of resources and modules should use underscores instead of dashes.